HCore-Cache/HCore-Cache.csproj

```diff
@@ -1,13 +1,14 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <TargetFramework>netcoreapp3.1</TargetFramework>
     <RootNamespace>HCore.Cache</RootNamespace>
     <AssemblyName>HCore.Cache</AssemblyName>
   </PropertyGroup>
 
-  <ItemGroup>
-    <PackageReference Include="Microsoft.Extensions.Caching.Redis" Version="2.2.0" />
+  <ItemGroup>    
+    <PackageReference Include="StackExchange.Redis.Extensions.AspNetCore" Version="6.3.6" />    
```

Comment:
Ideally we would go with version 8.0.5 (latest) and migrate to SystemTextJsonSerializer or the older NewtonsoftSerializer
However this could be feature effort due to cache usage changes that needs to be introduced as well.